### ISSUE TITLE
Modal Window - keyboard accessible

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#15]
**Requirement**: pop-ups, overlays, lightboxes, and other similar elements should be accessible to the keyboard and assistive technologies.

**Why is this required?** Access for the keyboard and assistive technologies need to be built in to such elements. This includes moving the keyboard and screenreader focus into the element, then supporting navigation and closure of the element. It also includes ensuring these elements are
visible and distinct under different colour and font size settings.

**Comment by papafed** [comment]
Can we try and eliminate popups and lightboxes?
Or at least on mobile, where they never work.
I think that in most cases we can use things like accordions to reveal content - for example, price plan details. 

It's only image galleries where you want to link to a larger version of an image that you might want a lightbox, but even then, it's hard to make them work well on mobiles.
